charts/kong/templates/_helpers.tpl

```diff
@@ -607,7 +613,24 @@ the template that it itself is using form the above sections.
 
 {{- $_ := set $autoEnv "KONG_PROXY_LISTEN" (include "kong.listen" .Values.proxy) -}}
 
-{{- $_ := set $autoEnv "KONG_STREAM_LISTEN" (include "kong.streamListen" .Values.proxy) -}}
+{{- $streamStrings := list -}}
+{{- if .Values.proxy.enabled -}}
+  {{- $tcpStreamString := (include "kong.streamListen" .Values.proxy) -}}
+  {{- if (not (eq $tcpStreamString "")) -}}
+    {{- $streamStrings = (append $streamStrings $tcpStreamString) -}}
+  {{- end -}}
+{{- end -}}
+{{- if .Values.udpProxy.enabled -}}
+  {{- $udpStreamString := (include "kong.streamListen" .Values.udpProxy) -}}
```

Comment:
can we use const, or env replace "kong.streamListen" here

Reply:
Not sure I follow--can you elaborate? How would you want this to function differently?

Reply:
Can you replace "kong.streamListen"  using Env "KONG_STREAM_LISTEN

Reply:
Still not sure I understand. `kong.streamListen` is a function this uses to generate a set of stream listen configurations (a comma-separated list of `IP:port arguments` strings) from the service configuration in values.yaml.

If you're asking if it can be overridden with user configuration, it can be--anything that we populate in `$autoEnv` can be overridden by setting it in the `env` section of values.yaml--though normally this wouldn't be, as it's pretty closely tied to all the rest of the auto-generated service configuration.